org.eclipse.jdt.ui/ui/org/eclipse/jdt/internal/ui/text/java/hover/JavadocHover.java

```diff
@@ -298,6 +299,15 @@ public void run() {
 		}
 	}
 
+	/**
+	 * @return timeout that we want to have on javadoc hover to avoid two browser instances hanging
+	 *         around for every opened java editor
+	 */
+	private static int getDisposeHoverTimeout() {
+		String key = IWorkbenchPreferenceConstants.DISPOSE_CLOSED_BROWSER_HOVER_TIMEOUT;
+		int timeout = Platform.getPreferencesService().getInt("org.eclipse.ui", key, 10, null);  //$NON-NLS-1$
```

Comment:
"10" is left from debugging. has to be changed to -1.

Reply:
`org.eclipse.ui/disposeClosedBrowserHoverTimeout=1000`

Reply:
Maybe I had the `-pluginCustomization` argument wrong, now I see its working. Will you be changing the default or should I?